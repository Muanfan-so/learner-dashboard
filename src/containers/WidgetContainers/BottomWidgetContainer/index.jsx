import React from 'react';

export const BottomWidgetContainer = () => <></>;

export default BottomWidgetContainer;
