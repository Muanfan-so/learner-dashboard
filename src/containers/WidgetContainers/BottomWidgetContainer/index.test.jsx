import { shallow } from 'enzyme';

import BottomWidgetContainer from '.';

describe('BottomWidgetContainer', () => {
  describe('snapshots', () => {
    test('default', () => {
      const wrapper = shallow(<BottomWidgetContainer />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
