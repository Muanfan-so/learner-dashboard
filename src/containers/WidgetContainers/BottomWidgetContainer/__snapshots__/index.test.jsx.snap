// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomWidgetContainer snapshots default 1`] = `<Fragment />`;
